docs/guides/primitives.mdx

```diff
@@ -81,32 +80,53 @@ sampler.run([
 ])
 ```
 
-## How to use Qiskit primitives
+## Benefits of Qiskit primitives
```

Comment:
this section and the one after it are still confusing to me, it's not clear if we're talking about the runtime primitives specifically or the general concept of primitives